### DISCUSSION TITLE
Errors with curl version when installing SDK tools

### DISCUSSION THREAD

**Discussion by eum602** [NillionNetwork/discussions#27] [Bugs]
Hi, hope this is the right place to raise an issue related to the way tools are installed:

When Installing SDK Tools I found some errors due to the `curl` version I have (7.68.0). The issue is presented since `--output-dir`  curl option is not availbable. A way I found to overcome this issue was to avoid the instruction and rather move to the temporal directory using `cd`. 

The change was just applied in `download` method which ended as follows:

```bash
download() {
    local download_dir="${1:?download_dir is required by download}"
    local download_url="${2:?download_url is required by download}"

    local current_dir=$PWD
    cd $download_dir
    if ! curl -L -O -s "$download_url"; then
        print_unexpected_error "Download of '$download_url' into '$download_dir' failed."
        return 1
    fi
    cd $current_dir
}
```

**Comment by jwolski2** [comment]
Hey @eum602 Thanks for the report. This is our bad. We'll get a fix out this week or next week. Stay tuned.

**Comment by jwolski2** [comment]
@eum602 The fix has been released. Can you do me a favor and confirm on your end that it works as expected?

**Reply by eum602** [reply]
Hi @jwolski2 yes I can confirm the issue has been addressed. Thank you for fixing it.
